micromasters_import/management/commands/queries/002_courserun_enrollment.sql

```diff
@@ -0,0 +1,65 @@
+--courserun migration from MicroMaster to MITxOnline
+INSERT INTO public.courses_courserun(
+    title,
+    courseware_id,
+    run_tag,
+    courseware_url_path,
+    start_date,
+    end_date,
+    enrollment_start,
+    enrollment_end,
+    expiration_date,
+    live,
+    created_on,
+    updated_on,
+    course_id)
+SELECT
+    mm_courserun.title,
+    mm_courserun.edx_course_key,
+    LEFT(CASE WHEN position('+' in mm_courserun.edx_course_key) > 0
+    THEN (regexp_match(mm_courserun.edx_course_key, '[^+]+$'))[1]
+    ELSE (regexp_match(mm_courserun.edx_course_key, '[^/]+$'))[1] END, 10) AS run_tag,
+    mm_courserun.enrollment_url,
+    mm_courserun.start_date,
+    mm_courserun.end_date,
+    mm_courserun.enrollment_start,
+    mm_courserun.enrollment_end,
+    mm_courserun.enrollment_end + INTERVAL '1 day',
+    NOT mm_courserun.is_discontinued,
```

Comment:
So we actually want this to be `true`, because `live` indicates whether or not the run shows up in the interface. It won't be enrollable because of the enrollment dates.